Server/package.json

```diff
@@ -41,7 +41,7 @@
     "@types/supertest": "^2.0.10",
     "@types/webpack-node-externals": "^2.5.0",
     "@typescript-eslint/eslint-plugin": "^5.8.1",
-    "@typescript-eslint/parser": "^4.12.0",
+    "@typescript-eslint/parser": "^5.8.1",

```

Comment:
Did you not need to update the react-test-renderer package in the calling package.json file?